p2p/status.go

```diff
@@ -19,59 +19,69 @@ package p2p
 import (
 	"time"
 
-	"github.com/ChainSafe/gossamer/common"
 	log "github.com/ChainSafe/log15"
 	"github.com/libp2p/go-libp2p-core/network"
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+// ExpireStatusInterval is the time between checking if status expired
+const ExpireStatusInterval = 20 * time.Minute
```

Comment:
is this from somewhere or arbitrary?

Reply:
arbitrary at the moment, the specification does not specify